bedrock/products/templates/products/vpn/desktop/index.html

```diff
@@ -0,0 +1,240 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "products/vpn/includes/macros.html" import hero_image, join_waitlist_button, vpn_nav_cta, vpn_content_well with context %}
+
+{% extends "products/vpn/base.html" %}
+
+{% block page_title_full %}Mozilla VPN: Protect Your Entire Device{% endblock %}
+{% block page_title_suffix %}{% endblock %}
+
+{% block page_desc %}Use the Mozilla VPN for full-device protection for all apps. With servers in 30+ countries, you can connect to anywhere, from anywhere.{% endblock %}
+
+{% block body_id %}mozilla-vpn-landing{% endblock %}
+
+{% set _utm_source = 'www.mozilla.org-vpn-product-page' %}
+{% set _utm_campaign = 'vpn-product-page' %}
+{% set _params = '?utm_source=' ~ _utm_source ~ '&utm_medium=referral&utm_campaign=' ~ _utm_campaign %}
+
+{% block page_css %}
+  {{ css_bundle('mozilla-vpn-landing') }}
+{% endblock %}
+
+{% block site_header %}
+  {% with
+    custom_nav_cta=vpn_nav_cta(
+      link_text='Try Mozilla VPN',
+      alt_link_text='Join the Waitlist',
+      sign_in_link_text='Sign In',
+      utm_source=_utm_source,
+      utm_campaign=_utm_campaign,
+    )
+  %}
+    {% include 'includes/protocol/navigation/navigation.html' %}
+  {% endwith %}
+{% endblock %}
+
+{% block content %}
+<main>
+  <section class="vpn-hero">
+    <div class="mzp-l-content mzp-t-content-xl">
+      <div class="vpn-hero-container">
+        <div class="vpn-hero-body">
+          <h1 class="vpn-hero-title">Mozilla VPN 2</h1>
+          <h2 class="vpn-hero-subtitle">{{ ftl('vpn-desktop-hero-title') }}</h2>
+          <div class="vpn-hero-desc">
+            <p>A VPN from the trusted pioneer in internet privacy.</p>
+
+            {{ vpn_subscribe_link(
```

Comment:
Not a blocker, just another flag: there's a good chance the styling on these subscription buttons is going to change at the macro level in the coming weeks. The changes may just be cosmetic (bringing them in closer alignment to Protocol buttons), but when the landing page design updates these may well need changing too.

Reply:
Should I fork the `landing.scss` styles to isolate the SEO pages from outside changes?

Reply:
The design for the main landing page is changing as of next week. I don't know exactly what those changes will be just yet, or how extensive - but I probably wouldn't rely on the current CSS for your pages here.

Reply:
The subscribe link has been removed from this page.